src/components/Pagination/Pagination.jsx

```diff
@@ -52,14 +52,16 @@ const Pagination = (props) => {
   );
 
   const renderPages = () => {
-    const pagination = [];
+    let pagination = [];
```

Comment:
We need to refactor this method: avoid `multiple else-if statements`, `let` and reduce the number of lines of code.